### PR TITLE
Color addition to FlatUIColors

### PR DESCRIPTION
**Pull request by avazquezpr**
# Modifications

Added new colors to the palette and fixed an issue with Swift 2.1.1
Created a file with Sketch that makes adding colors to the reference palette easier

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,29 +46,53 @@ The following is a list of all of the static color functions defined on `struct
...
+public static func galleryColor() -> OSColor!
+public static func ironColor() -> OSColor!
 public static func concreteColor() -> OSColor!
 public static func asbestosColor() -> OSColor!
+public static func pumiceColor() -> OSColor!
+public static func lynchColor() -> OSColor!
 ```
 
-![Colors Palette](https://github.com/brynbellomy/FlatUIColors/blob/master/ColorsPalette.png)
+![Colors Palette](https://github.com/avazquezpr/FlatUIColors/blob/master/ColorsPalette.png)
```
Comment (brynbellomy) [inline README.md]: Hey, would you mind changing this line back to point to this repo since it's the original?

**Comment by brynbellomy** [comment]
This is AWESOME!  Thanks so much for putting time and care into this PR.  I added one comment — if you don't have an objection to the change (for consistency's sake), I'd be more than happy to merge this in.

File: README.md (hunk 1 above)
Comment (avazquezpr) [inline README.md]: Of course! Will do when i get home :)

**Comment by avazquezpr** [comment]
Fixed. :+1:

**Comment by brynbellomy** [comment]
@avazquezpr Thanks again!  Just merged.

**Comment by brynbellomy** [comment]
@avazquezpr  FYI, new CocoaPods version just went out -> `0.3.0`

**Comment by avazquezpr** [comment]
@brynbellomy Great! :+1:

**Pull request merged**